package.json

```diff
@@ -3,13 +3,13 @@
   "devDependencies": {
     "@types/base64-js": "^1.2.5",
     "@types/flatbuffers": "^1.9.0",
-    "@types/prettier": "^1.13.2",
+    "@types/prettier": "=1.15.3",
     "@types/source-map-support": "^0.4.1",
     "@types/text-encoding": "0.0.33",
     "base64-js": "^1.3.0",
     "flatbuffers": "^1.9.0",
     "magic-string": "^0.22.5",
-    "prettier": "^1.14.0",
+    "prettier": "=1.15.3",
```

Comment:
why is it changed to an exact version? We should to check in the lockfile if we want to stick to one exact version.